css-display-3/Overview.bs

```diff
@@ -1047,6 +1047,28 @@ Appendix A: Glossary</h2>
 			<p class="example">
 				As another example, margins do not collapse across [=formatting context=] boundaries.
 
+			Specifications may call for an element or box to
+			<dfn export lt="establishes a formatting context | establish a formatting context | establish a new formatting context | establishes a new formatting context">establish a formatting context</dfn>
+			without further precision as to the type of formatting context or how to do so.
+			This operation is not applicable to
+			non-atomic inline-level boxes
+			nor to boxes with a layout-internal display-type.
```

Comment:
The correct terminology seems "display type" without hyphen. And I reiterate that display types should probably belong only to elements (and pseudo-elements), not boxes, but this problem is all over the spec.